packages/components/src/internal/components/user/UserLink.tsx

```diff
@@ -57,8 +57,14 @@ export const UserLink: FC<UserLinkProps> = props => {
 
     if (!isSelf && (!userCanReadUserDetails(user) || !targetUserDisplayValue)) {
         if (targetUserDisplayValue) return <span>{targetUserDisplayValue}</span>;
+
+        let title = 'User may have been deleted from the system.';
+        if (!user.isSystemAdmin) {
```

Comment:
Why this check for `!use.isSystemAdmin` here?

Reply:
Because if the user is site admin then they will likely find the user's details in the core.SiteUsers table. But a non site admin, might not find the suer because it was deleted or because perm was removed.